tests/2d_examples/test_2d_oscillating_beam/src/oscillating_beam.cpp

```diff
@@ -75,82 +63,44 @@ std::vector<Vecd> CreatBeamShape()
 
 	return beam_shape;
 }
-/**
-* @brief define the beam body
-*/
+//----------------------------------------------------------------------
+//	Bodies used in the case.
+//----------------------------------------------------------------------
 class Beam : public SolidBody
 {
 public:
 	Beam(SPHSystem &system, std::string body_name)
 		: SolidBody(system, body_name)
 	{
 		/** Geometry definition. */
-		std::vector<Vecd> beam_base_shape = CreatBeamBaseShape();
+		std::vector<Vecd> beam_base_shape = creatBeamBaseShape();
 		body_shape_ = new ComplexShape(body_name);
```

Comment:
when this is cleaned up?

Reply:
The cleanup is slow but I will do it steady.